devito/ir/support/guards.py

```diff
@@ -194,5 +214,6 @@ def transform_guard(expr, guard_type, callback):
     GuardBoundLe: GuardBoundGt,
     GuardBoundGt: GuardBoundLe,
     GuardBoundNextLe: GuardBoundNextGt,
-    GuardBoundNextGt: GuardBoundNextLe
+    GuardBoundNextGt: GuardBoundNextLe,
+    GuardOverflow: GuardOverflowLt
```

Comment:
I think for homogeneity we should have:

GuardOverflowGe: GuardOverflowLt
GuardOverflowLt: GuardOverflowGe

Reply:
fixed